bundles/ApplicationLoggerBundle/src/Maintenance/LogArchiveTask.php

```diff
@@ -63,6 +64,22 @@ public function execute(): void
         $sql = 'SELECT %s FROM '.ApplicationLoggerDb::TABLE_NAME.' WHERE `timestamp` < DATE_SUB(FROM_UNIXTIME('.$timestamp.'), INTERVAL '.$archive_threshold.' DAY)';
 
         if ($db->fetchOne(sprintf($sql, 'COUNT(*)')) > 0) {
+
+            $storageEngine = $this->config['applicationlog']['archive_db_table_storage_engine'];
+            if (empty($storageEngine)) {
```

Comment:
The $storageEngine variable always exists, so using empty() is not necessary.
```suggestion
            if (!$storageEngine) {
```